examples/json-to-record-with-projection/json_to_record_with_projection.md

```diff
@@ -0,0 +1,9 @@
+# JSON to record conversion with projection
+
+The `data.jsondata` library provides multiple APIs to selectively convert required fields from JSON data in the form of a `string`, `byte[]`, `byte-block-stream`, and `Ballerina JSON value` to a `Ballerina record`.
```

Comment:
It makes sense to quote keywords like `string` and `bytes[]`, however, I'm not sure if this applies to "Ballerina record".

Reply:
Updated https://github.com/ballerina-platform/ballerina-distribution/pull/5294/commits/1aba414e549730d8fc05d377816df6338bafd8e6